servicecontrol/license.md

```diff
@@ -8,12 +8,14 @@ tags:
 The following options outline how to add a license to ServiceControl.
 
 
-## ServiceControl Management
+## ServiceControl Management app
 
-ServiceControl Management  has a license user interface which will import the designated license file into the registry. The license file is added to the `HKEY_LOCAL_MACHINE` registry hive so it is available to all instances of ServiceControl regardless of the service account used.
+ServiceControl has a license graphical user interface which can be accessed in the ServiceControl Management app. The app is intalled together with ServiceControl and can be found in the Windows Start Menu.
 
-![](managementutil-addlicense.png 'width=500')
 
+The app will import the designated license file into the registry. The license file is added to the `HKEY_LOCAL_MACHINE` registry hive so it is available to all instances of ServiceControl regardless of the service account used.
```

Comment:
It was already there. But it seems useful to me, because it explains "it is available to all instances of ServiceControl regardless of the service account used"